Dockerfile

```diff
@@ -34,6 +34,15 @@ ENV IMAGE_NAME=saucelabs/stt-testcafe-node
 ARG BUILD_TAG
 ENV IMAGE_TAG=${BUILD_TAG}
 
+
+# Let saucectl know where to mount files
+RUN mkdir -p /home/seluser/__project__/ && chown seluser:seluser /home/seluser/__project__/
+LABEL com.saucelabs.project-dir=/home/seluser/__project__/
+ENV SAUCE_PROJECT_DIR=/home/seluser/__project__/
+
+# Let saucectl know what command to execute
+LABEL com.saucelabs.executable=/home/seluser/bin/testcafe
```

Comment:
Hm, rather than entrypoint? Is that so it's backwards compatible? (e.g. if this label is not available, it does it the old way?)

Reply:
Yes, that make totally sense, I will update that !

Reply:
@alexplischke  That's the idea !

The presence of this label allow to change the way to start the image.

New image (in the meaning of `executable image`) is defined by having this label.
So new saucectl will be able to catch it, and override `CMD` of the image by this executable (that will be kept as it to keep compatibility with older saucectl).

The old way of doing (side execution) would be remove later, in an upcoming breaking. Until this moment, cross compatibility should be achieved.